packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -477,6 +481,14 @@ class RetainingPathWidget extends StatelessWidget {
   }
 }
 
+String? _containerClass(ObjRef? object) {
```

Comment:
nit: maybe _objectClass or _containingClass, just to avoid confusion with the connotation of the flutter class Container

Reply:
Used _instanceClass, since it is specific to instanceRefs, not objects in general